plugins/coverbrowser.koplugin/listmenu.lua

```diff
@@ -245,7 +255,17 @@ function ListMenuItem:update()
 
         local bookinfo = BookInfoManager:getBookInfo(self.filepath, self.do_cover_image)
         if bookinfo and self.do_cover_image and not bookinfo.ignore_cover then
-            if not bookinfo.cover_fetched then
+            if bookinfo.cover_fetched then
+                -- trigger recalculation of thumbnail if size changed
+                if bookinfo.has_cover and bookinfo.cover_sizetag ~= "M" and bookinfo.cover_sizetag ~= cover_specs.sizetag then
+                    logger.dbg("ListMenuItem: thumbnail size changed from "
+                        .. (bookinfo.cover_sizetag or "nil") .. " to " .. (cover_specs.sizetag or "nil"))
```

Comment:
`logger.dbg("ListMenuItem: thumbnail size changed from ", bookinfo.cover_sizetag, " to ", cover_specs.sizetag)`
simpler and avoid string concatenation (on devices where we don't log debug).